libs/blocks/gnav/gnav.js

```diff
@@ -384,6 +384,30 @@ class Gnav {
     return searchBar;
   };
 
+  decorateAppsMenu = async (profileEl, gnav) => {
+    const appLauncherBlock = this.body.querySelector('.app-launcher');
+    if (!appLauncherBlock) return;
+    
+    gnav.classList.add('has-apps');
+    const appLauncher = await import('./gnav-appLauncher.js');
```

Comment:
@vhargrave I believe need the async on line# 387 for this.

Reply:
The await is for the import, because import returns a promise